Query Node unittest failed

<!-- Please state your issue using the following template and, most importantly, in English. -->

#### Steps/Code to reproduce:
historical unittest of query node in github action.

#### Expected result:
tests passed

#### Actual results:
```
--- FAIL: TestHistorical_GlobalSealedSegments (0.52s)
    historical_test.go:77: 
        	Error Trace:	historical_test.go:77
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestHistorical_GlobalSealedSegments
panic: runtime error: index out of range [0] with length 0 [recovered]
	panic: runtime error: index out of range [0] with length 0

goroutine 471 [running]:
testing.tRunner.func1.1(0x2671f80, 0xc000b7fd60)
	/usr/local/go/src/testing/testing.go:1076 +0x46a
testing.tRunner.func1(0xc000032f00)
	/usr/local/go/src/testing/testing.go:1079 +0x636
panic(0x2671f80, 0xc000b7fd60)
	/usr/local/go/src/runtime/panic.go:975 +0x3e9
github.com/milvus-io/milvus/internal/querynode.TestHistorical_GlobalSealedSegments(0xc000032f00)
	/go/src/github.com/milvus-io/milvus/internal/querynode/historical_test.go:78 +0xba5
testing.tRunner(0xc000032f00, 0x278cee0)
	/usr/local/go/src/testing/testing.go:1127 +0x203
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1178 +0x797
FAIL	github.com/milvus-io/milvus/internal/querynode	5.895s
FAIL
make: *** [test-go] Error 1
Makefile:115: recipe for target 'test-go' failed
2
Error: Process completed with exit code 2.
```

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others: 

#### Configuration file:


#### Additional context:

--- a/internal/querynode/historical_test.go
+++ b/internal/querynode/historical_test.go
@@ -66,13 +66,14 @@ func TestHistorical_GlobalSealedSegments(t *testing.T) {
 
 	// watch test
 	go n.historical.watchGlobalSegmentMeta()
+	time.Sleep(100 * time.Millisecond) // for etcd latency
 	segmentInfoStr := proto.MarshalTextString(segmentInfo)
 	assert.NotNil(t, n.etcdKV)
 	segmentKey := segmentMetaPrefix + "/" + strconv.FormatInt(segmentID, 10)
 	err := n.etcdKV.Save(segmentKey, segmentInfoStr)
 	assert.NoError(t, err)
 
-	time.Sleep(500 * time.Millisecond) // for etcd latency
+	time.Sleep(200 * time.Millisecond) // for etcd latency
 	segmentIDs = n.historical.getGlobalSegmentIDsByCollectionID(collectionID)
 	assert.Equal(t, 1, len(segmentIDs))
 	assert.Equal(t, segmentIDs[0], segmentID)

